sequence_processing_pipeline/templates/fastqc_job.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#SBATCH -J {{job_name}}
+#SBATCH -p {{queue_name}}
+#SBATCH -N {{node_count}}
+#SBATCH -n {{nprocs}}
+#SBATCH --time {{wall_time_limit}}
+#SBATCH --mem {{mem_in_gb}}G
+#SBATCH --array {{array_params}}
+set -x
+set +e
+date
+hostname
+echo ${SLURM_JOBID} ${SLURM_ARRAY_TASK_ID}
+cd {{output_path}}
+{% if modules_to_load is defined %}
+module load {{modules_to_load}}
+{% endif %}
+offset=${SLURM_ARRAY_TASK_ID}
+step=$(( $offset - 0 ))
```

Comment:
Isn't this effectively a noop?

Reply:
A good question. Your bash-fu was always better than mine. I will make an issue out of it and look into it.

Reply:
It is not okay to ask that bad code be merged. Please properly address the issues raised.

Reply:
It's literally the same bad code that works and has been driving multiqc for three years. It's already a part of FastQCJob - I just clipped it out from one file to a new file. We're going to hold up the release on this? It seems an awfully pedantic notion. I'll do it, but not tonight. It can wait until the morning.